src/documentation/pages/components/brand/examples/_black.vue

```diff
@@ -0,0 +1,81 @@
+</a>
+
+<!-- External Image Method -->
+<!-- Lumen -->
+<a class="chi-brand" href="/" aria-label="Lumen">
+  <img src="https://assets.ctl.io/images/lumen-logo-black.svg" alt="Lumen">
+</a>
+<!-- CenturyLink -->
+<a class="chi-brand -centurylink -black" href="/" aria-label="Lumen">
+  <img src="https://assets.ctl.io/images/centurylink-logo-black.svg" alt="CenturyLink">
+</a>`,
+      }
+    }
+  }
+})
+export default class Black extends Vue {
+  mounted() {
+    chi.tab(document.querySelectorAll('.chi-tabs-panel .chi-tabs'));
```

Comment:
Is this necessary? Is this for code snippet tabs?
If yes, there is no need for that as `ComponentExample` takes care of it.
Besides, In the previous example, I see the same. It will initialize all the tabs found on the page. Doing the same in different components will cause unpredictable side effects.

Reply:
Understood, I removed these mounted methods from Brand and Badge because it was redundant.

I tested from my side and everything is working fine! Thanks :)